sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -164,7 +164,7 @@ def _get_request_data(self, request):
         :return: The request data
         """
         if request.files:
-            form_data = aiohttp.FormData()
+            form_data = aiohttp.FormData(request.data or {})
```

Comment:
Does this behaviour now match what happens in a sync request?

Reply:
requests library accepts both data and files in request method. aiohttp only accepts data.

Reply:
@xiangyan99 is right. requests accepts both files and data and handles building the multipart/form-data request body for us. In aiohttp, we must make sure we include both into the `data` for them to do the same.